python/taichi/misc/image.py

```diff
@@ -43,8 +47,20 @@ def imdisplay(img):
         with BytesIO() as f:
             PIL.Image.fromarray(img).save(f, 'png')
             IPython.display.display(IPython.display.Image(data=f.getvalue()))
-    else:
-        ti.imshow(img)
+
+
+def imscale(img, w=512, h=None):
+    """
+    Scale a image into specific size.
+    """
+    if not isinstance(img, np.ndarray):
+        img = img.to_numpy()
+    if h is None:
+        h = w
+    u, v = (img.shape[0] - 1) / (w - 1), (img.shape[1] - 1) / (h - 1)
```

Comment:
How about adding an safety assertion before we use these `w` and `h`? like
```
w = int(w)
h = int(h)
assert w >=1 and h >=1, "xxx"
```
It can makes the error message more obvious.

Reply:
```suggestion
    assert isinstance(w, int) and isinstance(h, int) and w > 1 and h > 1
    u, v = (img.shape[0] - 1) / (w - 1), (img.shape[1] - 1) / (h - 1)
```